src/components/NewWorkspaceModal.js

```diff
@@ -160,7 +163,21 @@ const NewWorkspaceModal = Utils.withDisplayName('NewWorkspaceModal', ({
           placeholder: 'Select a billing project',
           value: namespace,
           onChange: ({ value }) => setNamespace(value),
-          options: _.uniq(_.map('projectName', billingProjects)).sort()
+          styles: {
+            option: provided => ({
+              ...provided,
+              padding: 0
+            })
+          },
```

Comment:
We could save some vertical space as single-lining it seems easy to read and short enough? 
```suggestion
          styles: { option: provided => ({ ...provided, padding: 0 }) },
```